client/components/shared/NewBankForm.js

```diff
@@ -9,14 +9,18 @@ export default class NewBankForm extends React.Component {
     constructor(props) {
         has(props, 'expanded');
         super(props);
-        this.state = {
-            expanded: this.props.expanded,
-            hasCustomFields: false,
-            customFields: []
-        };
+        let bank = store.getStaticBanks()[0];
+        this.state = Object.assign(
```

Comment:
I would rather not use Object.assign for such a simple use case. Can you do:

`this.state = this.computeState(bank);
this.state.expanded = this.props.expanded
`

instead, or is the linter forbidding this?